### ISSUE TITLE
[NPC] Knight Reinhard stuck in object

### ISSUE THREAD

**Issue by zulnam** [WarEmu/WarBugs#18569]
Description:
Knight Reinhard, required for Empire Epic Quest, is stuck in the boulder THE SILLY GOOSE.

STR:
1. In Chaos Wastes, go to 57k 18k.
2. Move next to the boulder emanating blue light.
3. Use shortcut to target Friendly units.

Expected:
Knight Reinhard should be reachable by players in order to interract with him for required quests.

Actual:
NPC is stuck in rock.

![image](https://user-images.githubusercontent.com/19802847/109527977-893b5100-7ab4-11eb-8cfa-818f8fdd427e.png)

**Comment by MaxHayman** [comment]
He's in his correct position on our test server. So once the server restarts he should be back (at most when we next patch). If you or anyone can find out how he got there in the first place please reopen.

**Comment by MaxHayman** [comment]
@Lheana Is this guy meant to go anywhere or does he stand static? Might be wroth setting his speed to 0 so he doesn't move?

**Comment by Lheana** [comment]
From what I saw he is stuck there because:
1- An order player pulls a Dragon Ogre nearby
2- The Dragon Ogre runs close enough to Reinhard to pull him
3- The Dragon Ogre follows the player running north, and Reinhard follow the Dragon Ogre
4- He is eventually stuck in this place

**Comment by zulnam** [comment]
Can confirm the position of Reinhard was reset after a server reboot.

I think the recently activated NPC pathing has caused a lot of friendly quest NPCs to be forced to move due to their stance and the stances of enemy mobs in the area. I have noticed for the elf epic quest as well that quest friendly NPCs were sometimes killed by nearby mobs and player had to wait for them to respawn. 

I know some NPCs in game have a neutral stance (not attacked or attackable by anybody). Not sure if they could share that stance and also be interactable for quest purpouses.

Closing ticket.
